docs/4.1/admin-guide.md

```diff
@@ -921,6 +921,32 @@ del`](cli-docs.md#tctl-tokens-rm) command:
 $ tctl tokens del 696c0471453e75882ff70a761c1a8bfa
 Token 696c0471453e75882ff70a761c1a8bfa has been deleted
 ```
+## Adding a node located behind NAT
+With the current setup you've only been able to add nodes that have direct access to the
+auth server and within the internal IP range of the cluster. We recommend
+setting up a [Trusted Cluster](/trustedclusters) if you have workloads split
+across different networks / clouds.
+
+Teleport Node Tunneling lets you add a node to an existing Teleport Cluster. This can be
+useful for IoT applications or for managing a couple of servers in a different network.  
+
+Similar to [Adding Nodes to Cluster](/quickstart/#adding-nodes-to-cluster), use `tctl` to
```

Comment:
should be linked to `quickstart.md#adding-nodes-to-cluster` (I'm trying to keep the internal links in a consistent format so that it's obvious where things go without any web server/mkdocs rewriting)